site/docs/v1/tech/core-concepts/applications.adoc

```diff
@@ -505,17 +505,17 @@ For example, if a password needs to be stored in an external configuration and t
 
 === Webhooks
 
+[WARNING.warning]
+====
+This feature is removed as of version 1.37.0.
+====
+
 The Webhooks tab allows you to select one or more webhooks to be used for this Application. In this example screenshot either no webhooks have been configured, or no application specific webhooks are configured.
```

Comment:
Should a version of this section be added to the Tenant core concepts page? I'm not sure whether there was additional webhook configuration added at the tenant level.

Reply:
In the upcoming PR